puppeteer-login-script.js

```diff
@@ -95,7 +95,7 @@ const createCollections = async function(context, page) {
     // eslint-disable-next-line no-console
     console.log('Creating Collections...');
     // eslint-disable-next-line dot-notation
-    await page.goto('http://127.0.0.1:8181/admin#/roles');
+    await page.goto('http://127.0.0.1:8181/admin#/roles', { waitUntil: 'networkidle0' });
     await page.waitFor(2000);
```

Comment:
Do we still need this? Or should we add longer wait times to be sure? Ditto in other places

Reply:
I think it should be okay to remove the wait times. However, I would prefer limiting this PR in scope to just adding the  `{ waitUntil: 'networkidle0' }` arg. I have another PR that will modify the puppeteer script more extensively https://github.com/oppia/oppia/pull/10152/files

Reply:
Okay